recipes/xilinx-runtime/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "xilinx-runtime" %}
+{% set version = "2.13.466" %}
+{% set quarter = "202210" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Xilinx/XRT/archive/refs/tags/{{ quarter }}.{{ version }}.tar.gz
+  sha256: d73d8366f6f6d66bfdcb67e72135f559a4d95644c5416c27dea79c235953fd20
+  patches:
+    - latest.patch
```

Comment:
Do you really need this huge patch? I'm reluctant to merge something which has a 150k+ loc patch.

Reply:
@xhochy I was worried about this. Basically, the large patch is moving our most recent release, to head. Then I apply the smaller patch, to make things work with conda. The head of our repository was much easier to get working with conda, and hence the second patch is much simpler.

However, I think I agree with you. It is not quite fair of me to ask that you revision control my 150k loc patch.

I'll optimize this.